packages/mdc-select/icon/README.md

```diff
@@ -0,0 +1,125 @@
+<!--docs:
+title: "Select Icon"
+layout: detail
+section: components
+excerpt: "Icons describe the type of input a select requires"
+iconId: text_field
+path: /catalog/input-controls/select/icon/
+-->
+
+# Select Icon
+
+Icons describe the type of input a select requires. They can also be interaction targets.
+
+## Design & API Documentation
+
+<ul class="icon-list">
+  <li class="icon-list-item icon-list-item--spec">
+    <a href="https://material.io/go/design-selects#selects-layout">Material Design guidelines: Text Fields Layout</a>
```

Comment:
I think maybe something got replaced here that shouldn't have - this URL doesn't exist